vms/volunteer/services.py

```diff
@@ -1,40 +1,51 @@
-from django.contrib.auth.models import User
 from django.core.exceptions import ObjectDoesNotExist
-from organization.services import *
+
+from organization.services import get_organization_by_name, get_organizations_ordered_by_name
 from volunteer.models import Volunteer
 
+
 def delete_volunteer(volunteer_id):
+    result = False
 
     volunteer = get_volunteer_by_id(volunteer_id)
 
     if volunteer:
-        #if the volunteer uploaded a resume, delete it as well
+        # if the volunteer uploaded a resume, delete it as well
         if has_resume_file(volunteer_id):
             delete_volunteer_resume(volunteer_id)
-        #Django docs recommend to set associated user to not active instead of deleting the user
+        # Django docs recommend to set associated user
+        # to not active instead of deleting the user
         user = volunteer.user
         user.is_active = False
-        #make a call to update the user
+        # make a call to update the user
         user.save()
-        #then delete the volunteer
+        # then delete the volunteer
         volunteer.delete()
+        result = True
     else:
-        return ObjectDoesNotExist
+        result = False
```

Comment:
@vubo I don't think that it is necessary to have an else clause since by the time one hits the return statement the result will be unchanged and false or delete occurred and True.